eng/scripts/RPMapping.json

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
Is this file necessary? Can we eliminate or default most of it? Having a separate file to maintain almost always causes issues and a higher cost to maintain.

Reply:
Removed the RPMapping file and do the RP folder name match by directly search readme in swagger repo.